frontend/components/commons/results/ResultsRecord.vue

```diff
@@ -18,14 +18,18 @@
 <template>
   <div v-if="dataset && viewSettings">
     <div
-      :class="annotationEnabled ? 'list__item--annotation-mode' : 'list__item'"
+      :class="
+        annotationEnabled && !pageSizeisOne
```

Comment:
1/ add a computed for annotationEnabled && !pageSizeisOne
2/ suggestion because it's a boolean : pageSizeisOne => isPageSizeEqualOne (camel case syntax means all the words start with uppercase)